Tnolet mac updates docs (#645)
checkly/checklyhq.com: site/content/docs/cli/npm-packages.md

---
title: Using NPM packages
weight: 5
menu:
  docs:
    parent: "CLI"
---

> Checkly lets you use JavaScript / TypeScript in your Browser checks and in the setup & teardown scripts you can

Reviewer: ```suggestion
Checkly lets you use JavaScript / TypeScript in your Browser checks and in API check setup & teardown scripts.
```

> add to your API checks. This JavaScript code executes in a runtime environment managed by Checkly.

Reviewer: ```suggestion
This JavaScript code executes in a runtime environment managed by Checkly.
```

> This environment has access to a specific Node.js versions and NPM packages.

Reviewer: ```suggestion
This environment has access to specific Node.js versions and NPM packages that are defined in [our runtimes](https://www.checklyhq.com/docs/runtimes/specs/).
```

> > This means not all NPM packages from NPM are available inside the context of a Check.

Reviewer: ```suggestion
> Not all NPM packages from NPM are available inside the context of a Check.
```
